tools/mason/MasonTest.chpl

```diff
@@ -64,25 +69,29 @@ private proc runTests(show: bool, run: bool) {
 
     // parse lockfile
     const toParse = open(projectHome + "/Mason.lock", iomode.r);
-    const lockFile = new Owned(parseToml(toParse));
+    const lockFile = parseToml(toParse);
 
     // Get project source code and dependencies
     const sourceList = genSourceList(lockFile);
     getSrcCode(sourceList, show);
     const project = lockFile["root"]["name"].s;
     const projectPath = "".join(projectHome, "/src/", project, ".chpl");
 
+    if isDir(joinPath(projectHome, "target/test/")) {
+      rmTree(joinPath(projectHome, "target/test/"));
+    }
     // Make target files if they dont exist from a build
     makeTargetFiles("debug", projectHome);
 
+    // get the test names from lockfile or from test directory
+    const testNames = getTests(lockFile, projectHome);
```

Comment:
> Why drop the delete-free Owned here?

Probably because this was causing issues. You can add the `Owned` back in above, and update `lockFile` to `lockFile.borrow()` here to avoid transferring ownership over to `getTests`